src/extension/background-script/actions/nostr/enable.ts

```diff
@@ -25,8 +25,22 @@ const enable = async (message: MessageAllowanceEnable, sender: Sender) => {
 
   const enabledFor = new Set(allowance?.enabledFor);
 
+  if (!isUnlocked) {
```

Comment:
Don't we have the same problem also for other providers? Should we change this behavior there as well?

Reply:
extended it for other providers